server/index.js

```diff
@@ -34,6 +35,12 @@ function main() {
     this.set('Timing-Allow-Origin', '*');
     yield next;
   });
+
+
+  app.use(cors({
```

Comment:
So could you just move this down onto the liveblog router or even the specific route.

Routes can take multiple middleware generators. eg

```javascript
router.get('/liveblog/thingy', cors(), thingy());
```